portal-impl/src/com/liferay/portal/service/impl/ResourcePermissionLocalServiceImpl.java

```diff
@@ -1201,6 +1210,98 @@ public void setResourcePermissions(
 			companyId, name, scope, primKey, 0, roleIdsToActionIds);
 	}
 
+	@Override
+	public void updateAction(
```

Comment:
Hey @drewbrokke, 

What do you think of changing the name of this method to better explain what it does. I find that the term update is very generic and not self explanatory. As an example, this method only covers one of two cases (selected=true|false) of the original one of the same name in RolesAdminPortlet.

Looking at other methods that do similar things they seem to be called addResourcePermission() I wonder if this one should just be called that or some variation of it.